### DISCUSSION TITLE
LMS 8.1.1 not running on DS412+ DSM6.2.4-25556

### DISCUSSION THREAD

**Discussion by aeq0805** [LMS-Community/slimserver#551] [General]
Hello... a newbie hope to get some assistance...

I've just installed LMS8.1.1 on my DS412+. Installation all done... but when I clicked Run, it tried running but then stopped as if I never clicked Run.

Tried uninstalling and reinstalling... No luck still.

Anyone mind helping? I have zero coding capability...

**Comment by mherger** [comment]
I don't know what package you installed. But please note that LMS on Synology DSM has reached the end. Synology stopped supporting it about a year ago, and pinkdot gave up maintaining LMS packages after that, as his work was based on Synology's. Depending on what architecture your DS is running on you might be able to run the Docker image (https://hub.docker.com/r/lmscommunity/logitechmediaserver).

**Comment by GLardner** [comment]
The latest update to Perl on Synology DSM has broken LMS for me. The Synology Package Installer acknowledges that it will break LMS. I don't know if there is any way to downgrade Perl on DSM or if there are any other dependencies in LMS that also will break it as LMS is no longer maintained.

For now, I am going to install LMS on a Raspberry Pi.

**Reply by StigNygaard** [reply]
It is still possible to uninstall Perl and LMS and then reinstall Perl 5.24 and PinkDot's LmsUpdate:
https://forums.slimdevices.com/showthread.php?113893-Request-for-Synology-to-upgrade-LMS-package-to-version-(8-x)&p=1015999&viewfull=1#post1015999

Even though it is not a long-term solution (no further updates to be expected), I will personally keep my LmsUpdate setup for now. I'm not ready to play with Docker configurations yet.

**Comment by aeq0805** [comment]
This is awesome. Thanks for brining my LMS back alive.

I thought abt Docker but got no good handle to learn for now... hopefully
we can get LMS hanging in there for as long as it could.

**Reply by faxm0dem** [reply]
I just had the same problem after upgrading perl.
I went another route : uninstall LMS on the syno, install docker, install the `lmscomunity-logitechmediaserver` docker image, change the ports to the regular ones, go!
Worked like a charm, and it won't break as it ships its own independant perl, container-style
